fedn/fedn/clients/reducer/templates/index.html

```diff
@@ -126,7 +126,7 @@ <h3 class="text-muted">{{client}}</h3>
                     {% if not h_latest_model_id %}
                     <!-- enctype="multipart/form-data" important for file upload-->
                     <form id="seed_upload_form" method="post" action="/seed" enctype="multipart/form-data">
-                        <input type="hidden" name="csrf_token" value="{{ csrf_token() }}"/>
```

Comment:
I think these serve a specific purpose, right @morganekmefjord . Should not be removed.

Reply:
Yes, should be important for security enhancements, even in the endpoints part I ignore them just to test the new integrated dashboard, if it's useful we can go further and extend it with CSRF, otherwise we ignore the dashboard and continue with own dashboard.